index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=\, initial-scale=1.0">
```

Comment:
- [ ] Your lighthouse linter is failing. I think you might have accidentally changed this line and it's causing it to fail. I tried locally and was able to fix this by changing it to:

`<meta name="viewport" content="width=device-width, initial-scale=1.0">`

So let's change this line with the above code please and try to run the linters again.